auth/src/desktop/rpcs/auth_request.cc

```diff
@@ -33,54 +35,22 @@ const char* kHeaderFirebaseLocale = "X-Firebase-Locale";
 
 AuthRequest::AuthRequest(::firebase::App& app, const char* schema,
                          bool deliver_heartbeat)
-    : RequestJson(schema) {
-  // The user agent strings are cached in static variables here to avoid
-  // dependencies upon other parts of this library.  This complication is due to
-  // the way the tests are currently configured where each library has minimal
-  // dependencies.
-
-  CheckEmulator();
-  static std::string auth_user_agent;           // NOLINT
```

Comment:
I'm guessing the user_agent stuff should not be getting removed

Reply:
I consulted with Shawn and they are actually not needed anymore. Include Shawn to review to confirm

Reply:
Most of the user-agent stuffs are not used, at least by Platform Logging pipeline, at all, except for the section I commented below. Therefore it is safe to remove.

However, I wonder if this should be in this particular PR. 

If you are planning to bundle this change along with emulator support, please make sure you include this information in the commit message, so it does not look like an accident change.

Reply:
done